include/sparrow/arrow_interface/arrow_array.hpp

```diff
@@ -34,23 +39,21 @@ namespace sparrow
      *                  for the the binary or utf-8 view type, which has one additional buffer compared to
      *                  the Columnar format specification (see Binary view arrays). Must be 0 or positive.
      * @param buffers Vector of sparrow::buffer
-     * @param children Vector of arrow_array_shared_ptr representing the children of the ArrowArray.
-     * @param dictionary arrow_array_shared_ptr or nullptr.
+     * @param children Raw vector of ArrowArray pointers or nullptr.
```

Comment:
`raw vector` is contradictory here, maybe `raw array`, `sequence`, `list` or `range`  would be best.
Same remark for `make_arrow_array_unique_ptr`.